src/Tribe/Admin/Organizer_Chooser_Meta_Box.php

```diff
@@ -50,7 +54,18 @@ protected function get_event( $event = null ) {
 	public function render() {
 		$this->render_dropdowns();
 		$this->render_add_organizer_button();
-		include $this->tribe->pluginPath . 'src/admin-views/new-organizer-meta-section.php';
+
+		if ( class_exists( 'Tribe__Events__Community__Main' ) ) {
```

Comment:
My only concern is introducing a Filter here only for this specific template, I would love to add something a little bit more broad but the current way the `getTemplateHierarchy` works would prevent me from doing it. I mean we have it duplicated on Tickets and Core, I'm not sure what would be the best approach here.

I think we could go for the `tribe_events_template_src/admin-views/new-organizer-meta-section.php` and filter it to be `community/modules/organizer-multiple.php`, but I felt this approach would look cleaner.